contributors/guide/contributor-cheatsheet/README.md

```diff
@@ -352,7 +357,7 @@ git checkout -b myfeature
 The main purpose of [squashing commits] is to create a clean readable git
 history or log of the changes that were made. Usually this is done in last
 phase of a PR revision. If you are unsure if you should squash your commits, it
-is better to err on the side of having more and leave it up to the judgement of
+is better to be on the side of having more and leave it up to the judgement of
```

Comment:
This might be an english thing, but err is right here. "To err on the side of caution"

Reply:
Reverting the same